### PR TITLE
Add bare-metal deployment guide

### PR DESCRIPTION
**Pull request by fmoor**

### CODE REVIEW

**Comment by fmoor** [comment]
This one is more vague than the other guides. I'm not sure how to be specific when the details are OS dependent.

**Review by 1st1 — COMMENTED** [review]
woot!

File: docs/guides/deployment/bare_metal.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Install the EdgeDB package.
+
+.. code-block:: bash
+
+   $ sudo yum install edgedb-1
+
+
+.. _ref_guide_deployment_bare_metal_enable_unit:
+
```
Comment (elprans) [inline docs/guides/deployment/bare_metal.rst:59]: Mention where the data is by default: `/var/lib/edgedb/1/data`

Comment (fmoor) [reply docs/guides/deployment/bare_metal.rst:59]: changed

**Review by elprans — APPROVED** [review]

**Pull request merged**